faq/index.html

```diff
@@ -251,7 +251,7 @@ <h3>Who gets the renewal fee?
         <h3>How do I transfer ownership of a name?
         <span><footer class='footer-caret-down' src='/img/footer/down-caret.svg' alt='Toggle expanded answer on mobile'/></span>
         <span class='hide'><footer class='footer-caret-up' src='/img/footer/up-caret.svg' alt='Toggle expanded answer on mobile'/></span></h3>
-        <span><p>If someone owns a name directly, the current owner can give the destination address/key to the new recipient. Sender creates a transaction to send the domain to receiver, and a block is mined on the blockchain. One week after the transaction is confirmed, it is locked in. Transferring ownership may also have payments embedded, so the recipient will receive coins if and only if the transfer is successful. This means that users do not need to use 3rd party escrow to pay for transfer. </p></span>
+        <span><p>If someone owns a name directly, the current owner can give the destination address/key to the new recipient. Sender creates a transaction to send the domain to receiver, and a block is mined on the blockchain. The transfer is then finalized by the sender after 288 blocks (~2 days). Transferring ownership may also have payments embedded, so the recipient will receive coins if and only if the transfer is successful. This means that users do not need to use 3rd party escrow to pay for transfer. </p></span>
```

Comment:
"If someone owns a name directly, the current owner can give the destination address/key to the new recipient."

I don't know what this is referring to actually, maybe we should remove it:

"The name owner sends a TRANSFER transaction to the receiver, and that transaction is confirmed in a block on the blockchain. After 288 blocks (~2 days) the name owner can send a FINALIZE transaction. Once that is confirmed, the new owner controls the name."

I guess we can leave the part about "may also have payments" - this is correct *now* that we have an atomic swap implementation. But at the time this page was written, it was merely conceptual ;-)

Reply:
>I don't know what this is referring to actually, maybe we should remove it

Which line are you referring to? I think both make sense. The first is simply handing over the private key or seed phrase for a wallet that has a name. The second is transferring on-chain to another address. Both are valid imo.

Reply:
Handing over a private key is incredibly bad practice - the original owner still has a copy of the key and so really, nothing has been transferred. I think this is a typo. The owner wouldn't give an "address" to a new recipient unless we're talking about a payment for the name. It's just unclear,

Reply:
Updated.